app/views/course_summaries/_course_summaries_table.js.jsx.erb

```diff
@@ -0,0 +1,104 @@
+<script type="text/jsx">
```

Comment:
This file seems to use tabs instead of spaces for indentation; please change to 2 space for indentation.